packages/api/src/external/hie/__tests__/set-doc-query-progress.test.ts

```diff
@@ -0,0 +1,354 @@
+import { DocumentQueryProgress } from "@metriport/core/domain/document-query";
```

Comment:
Decided to write unit tests for common scenarios specifically for the aggregate function